client/GameList.js

```diff
@@ -0,0 +1,81 @@
+import React, {useEffect, useState} from 'react';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import GameListItem from './GameListItem';
+import List from '@material-ui/core/List';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import axios from 'axios';
+import {withStyles} from '@material-ui/core';
+
+const styles = {
+    progress: {
+        marginLeft: 'calc(50% - 100px)',
+    },
+    failed: {
+        textAlign: 'center',
+    },
+};
+
+const GameList = ({match, location, classes}) => {
+    const {listType, listOption} = match.params;
+    const queryString = location.search;
+    const [gameList, setGameList] = useState([]);
+    const [isLoaded, setLoaded] = useState(false);
+
+    const getGameList = () => {
+        setGameList([]);
+        setLoaded(false);
+        axios.get(`/bgg/list/${listType}/${listOption}${queryString}`)
+            .then((response) => {
+                setGameList([...response.data]);
+                setLoaded(true);
+            }).catch(() => {
+            setLoaded(true);
+        });
+    };
+
+    useEffect(() => {
+        getGameList();
+    }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+    if (!gameList.length) {

```

Comment:
The ternary on [line 50](https://github.com/how-many-meeple/how-many-meeples/blob/173e6cdee185a920484a174bae94b9784ebee1d2/client/GameList.js#L50) suggests that `gameslist` might not exist.  If that's the case this line would error.

If you want to allow for that possibility you will need to check for its existence here, or earlier